b2b-sample/README.md

```diff
@@ -80,15 +85,7 @@ To create a user for Best Auto Mart with permissions to create an IdP for the **
 {
   "CommonConfig": {
     "AuthorizationConfig": {
-      "BaseOrganizationUrl": "<PARENT ORGANIZATION URL> ex: https://dev.api.asgardeo.io/t/guardio"
-    },
-    "ApplicationConfig": {
-      "SampleOrganization": [
-        {
-          "id": "<SUB ORGANIZATION ID>",
-          "name": "<SUB ORGANIZATION NAME>"
-        }
-      ]
```

Comment:
These configs are not required as user_organization claim is present in the ID token. The IS distributions with B2B feature where the user_organization claim is not presented can be affected due to removing this config. But we can suggest the previously released b2b-sample app for them to use, while keeping the latest b2b sample app up to date with latest changes without unnecessary configurations which can mislead the users.

Reply:
Initial release of the b2b app - https://github.com/wso2/samples-is/releases/download/v4.5.3/b2b-sample.zip

Reply:
Rather removing the configurations shall we update the readme with the proper description? Because as Sadil mentioned the sample will not work with IS distributions. This can be a problem when we release the GAs.

Reply:
The developers would ask why we need this configuration https://github.com/wso2/samples-is/pull/405/files#diff-34fe1b8cbee00497c7574755a2a49dc01937e2a04b08a16c2e7d3cfd24822197L88. Also I have seen some of them set the sub org id there, but it is not required. Hence why we can't remove that unnecessary config in order to not to confuse or answer back from us why that config stands for

Reply:
Yes, it was for backward compatibility, but we have previous b2b sample app releases which can be used for WSO2 PCC (until backed by IS 6.1). With these facts, still do we need that config in our latest sample. Please share the recommendations.